### PR TITLE
Add an empty requirements file

### PR DESCRIPTION
**Pull request by salmanmashayekh**

### CODE REVIEW

File: {{cookiecutter.repository_name}}/requirements.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+.
```
Comment (salmanmashayekh) [inline {{cookiecutter.repository_name}}/requirements.txt:1]: The docker image creation process expects a `requirements.txt` file to be available. It copies it to the docker and installs requirements based on that. It fails on `circleCI` if it does not exist.

**Review by salmanmashayekh — COMMENTED** [review]
The docker image creation process expects a requirements.txt file to be available. It copies it to the docker and installs requirements based on that. It fails on circleCI if it does not exist.

**Comment by pokey** [comment]
Interesting.  And what does that `requirements.txt` do?

**Comment by salmanmashayekh** [comment]
What do you mean @pokey ? It defines the package requirements for `pip`.

**Comment by pokey** [comment]
Yes but I was wondering what happens when you have a `requirements.txt` with only a `.`.  Never seen that before.  Assuming that will just install library and latest version of all dependencies, which is an odd thing for `requirements.txt` to do

@KensoDev any input on this?

**Comment by KensoDev** [comment]
@pokey @salmanmashayekh the `.` in `requirements.txt` is completely valid.
When we do dependency bump during the release, we generate a new one that is frozen.
Having the initial one with `.` is 👍

**Review by pokey — APPROVED** [review]

**Pull request merged**